cstore_reader.c

```diff
@@ -55,6 +56,10 @@ static StripeSkipList * LoadStripeSkipList(FILE *tableFile,
 										   StripeFooter *stripeFooter,
 										   uint32 columnCount,
 										   Form_pg_attribute *attributeFormArray);
+DeserializedColumnBlockData**
+CreateColumnBlockDataBuffers(uint32 columnCount, bool* columnMask, uint32 blockRowCount);
+void FreeColumnBlockDataBuffers(DeserializedColumnBlockData **deserializedColumnBlockDataArray,
+						   uint32 columnCount);
```

Comment:
1. First line exceeds the 90 character limit.
2. Second line not aligned with first line.